### PR TITLE
Update german translations

### PR DESCRIPTION
**Pull request by kripton**
I had to run "lupdate" to get the new strings into the translation files. This also updated the line numbers in the .ts-files. I've also commited all non-de_DE-files (middle commit). This is optional, feel free to drop that commit (only cherry-pick first and third commit) if you prefer.

### CODE REVIEW

File: plugins/dmxusb/src/DMX_USB_de_DE.ts [hunk 1]
```diff
@@ -79,62 +79,62 @@
...
         <location filename="dmxusbopenrx.cpp" line="158"/>
         <source>Patch this widget to a universe to find out.</source>
-        <translation type="unfinished">Dieses Geräte mit einem Universum verbinden um es herauszufinden.</translation>
+        <translation>Dieses Geräte mit einem Universum verbinden um es herauszufinden.</translation>
     </message>
     <message>
         <location filename="dmxusbopenrx.cpp" line="160"/>
         <source>System Timer Accuracy</source>
-        <translation type="unfinished">System-Timer Genauigkeit</translation>
+        <translation>System-Timer Genauigkeit</translation>
```
Comment (Lichtschalter-5000) [inline plugins/dmxusb/src/DMX_USB_de_DE.ts]: Why not "System-Uhr" or "System-Zeit"?

File: plugins/osc/OSC_de_DE.ts [hunk 2]
```diff
@@ -42,17 +42,17 @@
...
         <location filename="configureosc.ui" line="88"/>
         <source>Channel number calculator</source>
-        <translation type="unfinished"></translation>
+        <translation>Kanalnummer-Rechner</translation>
     </message>
     <message>
         <location filename="configureosc.ui" line="96"/>
         <source>OSC path</source>
-        <translation type="unfinished"></translation>
+        <translation>OSC pfad</translation>
```
Comment (Lichtschalter-5000) [inline plugins/osc/OSC_de_DE.ts]:
```suggestion
        <translation>OSC-Pfad</translation>
```

File: qmlui/qlcplus_de_DE.ts [hunk 3]
```diff
@@ -1720,124 +1720,124 @@ Ungesicherte Änderungen gehen verloren, wenn du sie nicht speicherst.</translat
...
         <location filename="qml/fixturesfunctions/PaletteManager.qml" line="121"/>
         <source>Position</source>
-        <translation type="unfinished">Position</translation>
+        <translation>Position</translation>
     </message>
     <message>
         <location filename="qml/fixturesfunctions/PaletteManager.qml" line="134"/>
         <source>Delete the selected palette(s)</source>
-        <translation type="unfinished"></translation>
+        <translation>Ausgewählte Palette löschen</translation>
```
Comment (Lichtschalter-5000) [inline qmlui/qlcplus_de_DE.ts]:
```suggestion
        <translation>Ausgewählte Palette(n) löschen</translation>
```

File: ui/src/qlcplus_de_DE.ts [hunk 4]
```diff
@@ -4926,7 +4926,7 @@ Dauer: %3
         <location filename="showmanager/showitem.cpp" line="70"/>
         <source>Click to move this item along the timeline</source>
         <oldsource>Click to move this item across the timeline</oldsource>
-        <translation type="unfinished">Klicken um das Element über die Zeitleiste zu bewegen</translation>
+        <translation>Klicken um das Element über die Zeitleiste zu bewegen</translation>
```
Comment (Lichtschalter-5000) [inline ui/src/qlcplus_de_DE.ts]:
```suggestion
        <translation>Klicken, um das Element über die Zeitleiste zu bewegen</translation>
```

File: qmlui/qlcplus_de_DE.ts [hunk 5]
```diff
@@ -3713,27 +3713,27 @@ Zugriffsstufe:</translation>
     <message>
         <location filename="qml/virtualconsole/VCCueListProperties.qml" line="149"/>
         <source>Side fader</source>
-        <translation type="unfinished"></translation>
+        <translation>Seitenfader</translation>
```
Comment (Lichtschalter-5000) [inline qmlui/qlcplus_de_DE.ts]: "Side fader" translated to "Seitenfader"

File: ui/src/qlcplus_de_DE.ts [hunk 6]
```diff
@@ -5734,7 +5734,7 @@ Dauer: %3
         <location filename="virtualconsole/vccuelistproperties.ui" line="276"/>
         <source>Side Fader</source>
         <oldsource>Side Faders</oldsource>
-        <translation type="unfinished">Seiten-Schieberegler</translation>
+        <translation>Seiten-Schieberegler</translation>
```
Comment (Lichtschalter-5000) [inline ui/src/qlcplus_de_DE.ts]: "Side Faders" translated to "Seiten-Schieberegler".
Stick to one, preferrably the first one.

File: plugins/dmxusb/src/DMX_USB_de_DE.ts (hunk 1 above)
Comment (kripton) [reply plugins/dmxusb/src/DMX_USB_de_DE.ts]: I'd translate `Uhr` with `clock` and `Zeit` with `time`. `Timer` is harder to translate. And `System-Timer Genauigkeit` is also used as this in two other locations.

**Comment by kripton** [comment]
@Lichtschalter-5000 : Thanks for the review, most suggestions have been applied :sunglasses:

**Comment by mcallegari** [comment]
Thanks guys. Please let me know when this is good to go

**Comment by kripton** [comment]
> Thanks guys. Please let me know when this is good to go
From my side, translations are good and done.
@mcallegari: If you are okay with merging all the changes done by "lupdate", feel free to merge

**Comment by mcallegari** [comment]
@kripton I'm OK with that. In any case I would have done it myself sooner or later.
Thanks

**Pull request merged**